GitUI/CommandsDialogs/FormArchive.Designer.cs

```diff
@@ -260,7 +260,7 @@ private void InitializeComponent()
             | System.Windows.Forms.AnchorStyles.Left) 
             | System.Windows.Forms.AnchorStyles.Right)));
             this.labelMessage.AutoEllipsis = true;
-            this.labelMessage.Font = new System.Drawing.Font("Microsoft Sans Serif", 8.25F, System.Drawing.FontStyle.Bold, System.Drawing.GraphicsUnit.Point, ((byte)(0)));
+            this.labelMessage.Font = new System.Drawing.Font(this.labelMessage.Font, System.Drawing.FontStyle.Bold);
```

Comment:
there are a few changes like this, won't they be replaced by the Designer, if/when the dialog is modified in the designer?